app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
The class must inherit from the ABC also, the same about Elf

Reply:
I don't understand why, because Dwarf and Elf already abstract classes, they are inherited from the Player class, which is inherited from ABC.